src/clr-angular/utils/_overwrites.clarity.scss

```diff
@@ -71,7 +67,6 @@ $clr-content-area-padding-left: null; // $clr-baseline*1;
 $clr-card-bg-color: null; // $clr-color-neutral-0
 $clr-card-border-color: null; // $clr-color-neutral-200
 $clr-card-box-shadow: null; // 0 $clr_baselineRem_0_125 0 0 rgba(215, 215, 215, 1)
-$clr-card-border: null; // $clr-global-borderwidth solid rgba(215, 215, 215, 1)
```

Comment:
Is this no longer an overwrite-able attribute?

Reply:
`$clr-card-border-color` is overwritable. the border width is tied to `$clr-global-borderwidth`. and the border type is just `solid`.

with borders and some other styles, it makes sense to destructure these mash-up styles.